packages/base/src/widget.ts

```diff
@@ -67,6 +69,17 @@ export class JupyterCadPanel extends SplitPanel {
     SplitPanel.setStretch(this._jupyterCadMainViewPanel, 1);
   }
 
+  private _syncEditor(tracker: WidgetTracker<IJupyterCadWidget>) {
```

Comment:
Nitpicking suggestion:

I would not define a method for this as this is called just once in the constructor. So maybe just inline that code there :)

Reply:
Ah right! I agree. Sorry Arjun for the misleading comments earlier

Reply:
No worries, It's been a good mental exercise 🤣 
I'll be on it.

Reply:
Done in [`4115b79`](https://github.com/jupytercad/JupyterCAD/pull/448/commits/4115b79a5cb1a5d636ed4982df147d465364c8c4)
And I believe I myself am responsible that this PR took longer than usual as my initial iterations of solution were not close to perfect.